DataPreprocessing.ipynb

```diff
@@ -128,12 +144,18 @@
       "Starting on file Type_2\n",
       "10%...20%...30%...40%...50%...60%...70%...80%...90%...100%\n",
       "Starting on file Type_3\n",
-      "10%...20%...30%...40%...50%...60%...70%...80%...90%...100%\n"
+      "10%...20%...30%...40%...50%...60%...70%...80%...90%...100%\n",
+      "Starting on file Type_1\n",
+      "10%...20%...30%...40%...50%...60%...70%...Error resizing image 5893.jpg in D:\\Work\\Kaggle\\Cancer Screening\\Cancer-Screening-NN\\Data\\additional_Type_1_v2\\Type_1\n",
+      "80%...90%...100%\n",
+      "Starting on file Type_2\n"
      ]
     }
    ],
    "source": [
-    "train_data = create_train_data(IMG_SIZE)\n",
+    "train_folders = [cte_train_dir, cte_additional_t1_dir, cte_additional_t2_dir, cte_additional_t3_dir]\n",
```

Comment:
Also are we not labeling the additional training images for some reason. You know, the images that exist outside of everything because Chris is a neurotic neurotic soul?